README.md

```diff
@@ -166,22 +166,73 @@ For more information on test writing, please visit our [documentation](http://su
-- `--conflict OPTION` - use the `abort` option to abort any runs in progress in the same environment as your new run. use the `abort-all` option to abort all runs in progress.
-- `--fg` - results in the foreground - rainforest-cli will not return until the run is complete. This is what you want to make the build pass / fail dependent on rainforest results
-- `--wait RUN_ID` - wait for an existing run to finish instead of starting a new one, and exit with a non-0 code if the run fails. rainforest-cli will exit immediately if the run is already complete.
-- `--fail-fast` - fail the build as soon as the first failed result comes in. If you don't pass this it will wait until 100% of the run is done. Use with `--fg`.
-- `--custom-url` - use a custom url for this run. Example use case: an ad-hoc QA environment with [Fourchette](https://github.com/rainforestapp/fourchette). You will need to specify a `site_id` too for this to work. Note that we will be creating a new environment for this particular run.
-- `--git-trigger` - only trigger a run when the last commit (for a git repo in the current working directory) has contains `@rainforest` and a list of one or more tags. E.g. "Fix checkout process. @rainforest #checkout" would trigger a run for everything tagged `checkout`. This over-rides `--tag` and any tests specified. If no `@rainforest` is detected it will exit 0.
-- `--description "CI automatic run"` - add an arbitrary description for the run.
-- `--embed-tests` - Use with `rainforest export` to export your tests without extracting the
-steps of an embedded test.
-- `--test-folder /path/to/directory` - Use with `rainforest [new, upload, export]`. If this option is not provided, rainforest-cli will, in the case of 'new' create a directory, or in the case of 'upload' and 'export' use the directory, at the default path `./spec/rainforest/`.
+<br>
+###Browsers
+specify the browsers you wish to run against. This overrides the test own settings. Valid browsers can be found in your account settings.
+- <pre>--browsers <b>browser_name</b>[<b>,browser_name</b>]</pre>
+###Tags
+ only run tests which have this tag (recommended if you have lots of [test-steps](http://docs.rainforestqa.com/pages/example-test-suite.html#test_steps))!
+<pre>--tags <b>tag</b>[<b>,tag,...</b>]</pre> 
 
+###Site-ID
+ only run tests for a specific site. Get in touch with us for help on getting that you site id if you are unable to.
+
+<pre>--site-id <b>ID</b></pre>
+
+###Folder-ID
+ run tests in specified folder.
+<pre>--folder <b>ID</b></pre>
+
+###Environment-ID
+run your tests using this environment. Otherwise it will use your default environment
+<pre>--environment-id <b>ID</b></pre>
+
+###Crowd
+<pre>--crowd [<b>default</b>|<b>on_premise_crowd</b>]</pre>
+
+
+###Conflict
+use the abort option to abort any runs in progress in the same environment as your new run. use the abort-all option to abort all runs in progress.
+- <pre>--conflict <b>option</b></pre> </pre>
+
+
+###Fg
```

Comment:
I thought it would be better to use the command names as headers. Is it better to just use Foreground?